cmd/fleetctl/query.go

```diff
@@ -138,6 +139,12 @@ func queryCommand() *cli.Command {
 				if strings.Contains(err.Error(), "no hosts targeted") {
 					return errors.New(fleet.NoHostsTargetedErrMsg)
 				}
+				if strings.Contains(err.Error(), fleet.InvalidLabelSpecifiedErrMsg) {
+					pattern := fmt.Sprintf("(%s.*)$", regexp.QuoteMeta(fleet.InvalidLabelSpecifiedErrMsg))
```

Comment:
What's the issue with returning the error as-is?

Reply:
the error that is returned looks like this:
```
create live query: POST /api/latest/fleet/queries/run_by_identifiers received status 400 Bad request: [iamnotalabel] are not valid label names, remove to continue
```

Doesn't seem to match the nicely formatted errors return by most of the fleetctl commands